server/routes/bookVisit/visitBookedController.ts

```diff
@@ -9,7 +9,7 @@ export default class VisitBookedController {
       res.render('pages/bookVisit/visitBooked', {
         bookingConfirmed,
         prison: bookingConfirmed.prison,
-        showServiceNav: true,
+        showOLServiceNav: true,
```

Comment:
Renaming this to distinguish between the **GOVUK One Login** service navigation and the standard **GOVUK service navigation** 😵‍💫